### ISSUE TITLE
template error when using --binary-file

### ISSUE THREAD

**Issue by wjackson** [bojand/ghz#284]
**Proto file(s)**
Anything with a `bytes` field. [BytesValue](https://github.com/protocolbuffers/protobuf/blob/5ab8239d4b7da3589d3660233e9a2a7e89364083/src/google/protobuf/wrappers.proto#L122) for example.

**Command line arguments / config**
`--binary-file`

**Describe the bug**

Binary protobuf messages with bytes fields cause template errors like this:

    template: call_template_data:2: unrecognized character in action: U+FFFD '�'

If the bytes happen to contain character sequences that look like `template` directives.

The problem `template` call seems to be [here](https://github.com/bojand/ghz/blob/master/runner/calldata.go#L131)

**To Reproduce**

Here's a [test](https://github.com/wjackson/ghz/blob/3bf8309d678026813fd43a501819c17008169f69/runner/run_test.go#L460]) that produces the error.

**Expected behavior**

Binary messages provided via `--binary` or `--binary-file` are not processed by `template` and work regardless of what byte patterns they happen to have.

**Environment**
 - OS: linux
 - ghz: 0.95.0

**Comment by bojand** [comment]
Hmm I am a little confused by the steps to reproduce as it's using `greeter.proto` file.

if we have a service definition like:

```proto
syntax = "proto3";

package wrapped;

option go_package = "internal/wrapped";

import "google/protobuf/wrappers.proto";

service WrappedService {
  rpc GetBytesMessage (google.protobuf.BytesValue) returns (google.protobuf.BytesValue);
}
```

This works:

```go
protoMsg := wrappers.BytesValue{Value: []byte("Hello World!")}
msg, err := proto.Marshal(&protoMsg)

report, err := Run(
	"wrapped.WrappedService.GetBytesMessage",
	WithProtoFile("../testdata/wrapped.proto", []string{"../testdata"}),
	WithBinaryData(msg),
	// ...
)
```

Similarly we can write the data to a fine and use `--binary-file` option.

**Comment by wjackson** [comment]
As mentioned, the problem happens when the byte sequence, by bad luck, contains something that looks like a `template` directive. `[]byte("Hello World!")` doesn't exercise the bug. The test I linked to does though.

**Comment by bojand** [comment]
Hello, sorry I misunderstood the issue... This should be fixed in [v0.98.0](https://github.com/bojand/ghz/releases/tag/v0.98.0). If there are more problems with this feel free to reopen or create a new issue.
